### PR TITLE
NONE:Fix jmeter_open_editor and selenium_a_login test

### PR DESCRIPTION
**Pull request by mmizin**

### CODE REVIEW

File: app/jmeter/confluence.jmx [hunk 1]
```diff
@@ -7836,13 +7841,13 @@ vars.put(&quot;p_new_file_name&quot;, (new Random().with {(1..9).collect {((&apo
                 <hashTree/>
                 <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion" enabled="true">
                   <collectionProp name="Asserion.test_strings">
-                    <stringProp name="175358039">title=&quot;Preview&quot;</stringProp>
+                    <stringProp name="2046823756">&lt;title&gt;Edit</stringProp>
```
Comment (ometelytsia) [inline app/jmeter/confluence.jmx]: What does this name mean?

File: app/jmeter/confluence.jmx [hunk 2]
```diff
@@ -7836,13 +7841,13 @@ vars.put(&quot;p_new_file_name&quot;, (new Random().with {(1..9).collect {((&apo
...
                     <stringProp name="1975472560">Update&lt;/button&gt;</stringProp>
                   </collectionProp>
                   <stringProp name="Assertion.custom_message"></stringProp>
                   <stringProp name="Assertion.test_field">Assertion.response_data</stringProp>
                   <boolProp name="Assertion.assume_success">false</boolProp>
-                  <intProp name="Assertion.test_type">16</intProp>
+                  <intProp name="Assertion.test_type">2</intProp>
```
Comment (ometelytsia) [inline app/jmeter/confluence.jmx]: What does this mean?

File: app/selenium_ui/confluence/modules.py [hunk 3]
```diff
@@ -217,4 +220,4 @@ def _wait_until(webdriver, expected_condition, interaction, time_out=timeout):
         message += (f"Timed out after {time_out} sec waiting for {str(expected_condition)}. \n"
                     f"Locator: {expected_condition.locator}")
 
-    return WebDriverWait(webdriver, time_out).until(expected_condition, message=message)
+    return WebDriverWait(webdriver, time_out).until(expected_condition, message=message)
```
Comment (ometelytsia) [inline app/selenium_ui/confluence/modules.py]: Python file should have empty line in the end the file.

File: app/selenium_ui/confluence/modules.py [hunk 4]
```diff
@@ -38,20 +38,23 @@ def measure(webdriver, interaction):
...
-                        interaction).click()
-            _wait_until(webdriver, EC.element_to_be_clickable((By.CSS_SELECTOR, '.aui-button-link')),
-                        interaction).click()
-            spaces = _wait_until(webdriver, EC.visibility_of_any_elements_located(
-                (By.CSS_SELECTOR, '.intro-find-spaces-space>.space-checkbox')), interaction)
-            if spaces:
-                spaces[0].click()
-            _wait_until(webdriver, EC.element_to_be_clickable((By.CSS_SELECTOR, '.intro-find-spaces-button-continue')),
-                        interaction).click()
+            if webdriver.find_element_by_class_name('grow-aui-progress-tracker-step-current').text == 'Welcome':
```
Comment (ometelytsia) [inline app/selenium_ui/confluence/modules.py]: Could you please format this code more consistent.

File: app/jmeter/confluence.jmx (hunk 1 above)
Comment (mmizin) [reply app/jmeter/confluence.jmx]: I think this name="175358039" JMeter uses under the hood.

File: app/jmeter/confluence.jmx (hunk 2 above)
Comment (mmizin) [reply app/jmeter/confluence.jmx]: I think it is the type of test type for e.g "Contains" or "Sub string"

**Pull request merged**